Applications/FileIO/GocadIO/GocadAsciiReader.cpp

```diff
@@ -27,48 +28,91 @@ namespace Gocad
 const std::string mat_id_name = "MaterialIDs";
 const std::string eof_error = "Error: Unexpected end of file.";
 
-GocadTSurfaceReader::GocadTSurfaceReader()
+GocadAsciiReader::GocadAsciiReader()
+: _export_type(GocadDataType::ALL) {}
+
+GocadAsciiReader::GocadAsciiReader(GocadDataType const t)
```

Comment:
```suggestion
GocadAsciiReader::GocadAsciiReader(GocadDataType const t = GocadDataType::ALL)
```
... then the default ctor is not needed.

Reply:
:heavy_check_mark: